### DISCUSSION TITLE
@nrwl/next:server very slow booting time

### DISCUSSION THREAD

**Discussion by AtlasRW** [nrwl/nx#12902] [General]
Hi everyone,
I wanted to talk about the @nrwl/next package and how, after migrating my NextJS app into an Nx monorepo, I noticed a VERY significant drop of performance while booting a server, whether it is in development with Hot Module Reload or production with the build output.
Booting time changed from ~10s to ~2mn before the first page load from the server is available.

Does anyone have noticed the same changes ? And does anyone know why ?
I guess that is linked to Nx's custom NextJS server, but I'm not sure.

**Comment by nickcarnival** [comment]
Same issue, I've gone from less than 500ms fast refreshes to 40000ms fast refreshes.
